source/Halibut.Tests/CancellationViaClientProxyFixture.cs

```diff
@@ -156,9 +95,12 @@ await using (var clientAndService = await clientAndServiceTestCase.CreateTestCas
                 // Give time for the cancellation to do something
                 await Task.Delay(TimeSpan.FromSeconds(2), CancellationToken);
 
-                (await AssertException.Throws<Exception>(inFlightRequest))
-                    .And
-                    .Should().Match<Exception>(x => x is OperationCanceledException || (x.GetType() == typeof(HalibutClientException) && x.Message.Contains("The ReadAsync operation was cancelled")));
+#pragma warning disable VSTHRD003
```

Comment:
I think we can get away with not disabling this warning.

If we use the override that just takes the task, that should fix it? E.g.:

```
(await AssertionExtensions.Should(inFlightRequest).ThrowAsync<Exception>())
```

Reply:
Ah nice, I just hammered in the ignores 😆